Mathlib/Algebra/Group/Commute/Defs.lean

```diff
@@ -258,19 +258,20 @@ end Commute
 section Monoid
 variable {M : Type*} [Monoid M] {n : ℕ}
 
+-- Note (ACL) : this one has been changed to the initial meaning of pow_succ
```

Comment:
The `#align`s should be updated to reflect this. This comment should be removed; it's useful to a PR reviewer, but not to someone who finds the code later; so it belongs in the PR not the code!

Reply:
```suggestion
```